moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -603,7 +604,7 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
     return locations;
   }
 
-  MoveItErrorCode place(const moveit_msgs::PlaceGoal& goal)
+  MoveItErrorCode place(const moveit_msgs::PlaceGoal& goal) const
```

Comment:
calls ROS action

Reply:
I believe you commented on this issue in the general feedback section above. The thinking here was that since the unique_ptr `pick_action_client_` is not modified, then constness of the member variables of `move_group_interface` is satisfied and thus the method could be marked as const. However, taking a closer look at this issue and noticing that calling the method _does_ change the state of the object owned by the unique_ptr, I feel it is easy to argue also the opposite. To sum it up, I think this could be seen as a limitation of C++ (compiler should IMO warn when the internal state of the pointed object is modified by a const method) and bit of a grey area. Having said that I will fix this and remove const since it is a bit misleading here.

Reply:
`const` is still there.